plugins/source/aws/codegen/main.go

```diff
@@ -103,13 +104,12 @@ func generateResources() ([]*recipes.Resource, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to set parent-child relationships: %w", err)
 	}
+	grp := new(errgroup.Group)
```

Comment:
This change seems unrelated to this PR. Can we extract it to a separate one? Also it would be useful to see some benchmarks if this improves the codegen performance